contrib/!lang/python/packages.el

```diff
@@ -162,16 +162,24 @@
 
       (defun python-setup-shell ()
         (if (executable-find "ipython")
-            (setq python-shell-interpreter "ipython"
-                  ;; python-shell-interpreter-args (if (system-is-mac)
-                  ;;                                   "--gui=osx --matplotlib=osx --colors=Linux"
-                  ;;                                 (if (system-is-linux)
-                  ;;                                     "--gui=wx --matplotlib=wx --colors=Linux"))
-                  python-shell-prompt-regexp "In \\[[0-9]+\\]: "
-                  python-shell-prompt-output-regexp "Out\\[[0-9]+\\]: "
-                  python-shell-completion-setup-code "from IPython.core.completerlib import module_completion"
-                  python-shell-completion-module-string-code "';'.join(module_completion('''%s'''))\n"
-                  python-shell-completion-string-code "';'.join(get_ipython().Completer.all_completions('''%s'''))\n")
+            (if (system-is-mswindows)
+                (setq python-shell-interpreter "python"
+                      python-shell-interpreter-args "-i c:\\python27\\scripts\\ipython-script.py"
```

Comment:
I agree. I should have introduced a variable to hold the real path of  ipython-script.py. However, I'm not familiar with elisp since I'm new to it. Can you help me?